beacon.yaml

```diff
@@ -1621,30 +1621,30 @@ components:
       example: GRCh38
     ReferenceBases:
       description: |
-        Reference bases for this variant (starting from `start`). 
-        Accepted values: [ACGTN]*. N is a wildcard, that denotes the 
-        position of any base, and can be used as a standalone base of any 
-        type or within a partially known sequence. For example a sequence 
-        where the first and last bases are known, but the middle portion can 
-        exhibit countless variations of [ACGT], or the bases are unknown: 
-        ANNT the Ns can take take any form of [ACGT], which makes both ACCT 
+        Reference bases for this variant (starting from `start`).
+        Accepted values: [ACGTN]*. N is a wildcard, that denotes the
+        position of any base, and can be used as a standalone base of any
+        type or within a partially known sequence. For example a sequence
+        where the first and last bases are known, but the middle portion can
+        exhibit countless variations of [ACGT], or the bases are unknown:
+        ANNT the Ns can take take any form of [ACGT], which makes both ACCT
         and ATGT (or any other combination) viable sequences.
       type: string
-      pattern: '^([ACGTN]+)$'
+      pattern: '^([ACGTN]*)$'
```

Comment:
Can you elaborate the need of this change? If the user specifies this field, let's ensure it contains a (valid) value with `+`. If they want it empty, just leave it out.

Reply:
mmm I see. But the change was done in `referenceBases` field, not in `alternateBases`, which still contains `+`. So, if I got it right, the change (`+` to `*`) has been applied to the wrong place.

Reply:
Please, @babisingh, let us know what you think about this.

Reply:
Hi, @sdelatorrep I agree with  Michael's suggestion of 'using clearly defined variantType values for queries'. So perhaps it looks something like `100: CT > DEL` when user is not sure about `alternateBases`

- About the ` (+ to *)` why it cannot be used in both `referenceBases` and `alternateBases` in case of searching for long deletions, for example:` CAA**ATACG** > AT*`  instead to search for `CAA**ATACG** > ATACG`, just a speculation.

-

Reply:
How does VRS solve the "emptyness" (it is quite weird)? 
How do they manage the insertions? the same way? "nothing" to "C" (e.g.)?

Reply:
@jrambla Yes. Good documentation now, e.g. https://vrs.ga4gh.org/en/stable/impl-guide/normalization.html?highlight=insertion#allele-normalization:

> If reference allele sequence is empty, the Allele represents an insertion in the reference. If alternate allele sequence is empty, the Allele represents a deletion in the reference.

... i.e. trim all the overlapping bases. This was confirmed with @ahwagner; and @rahelp will write some documentation/examples. But the "no anchor bases" will stand.

Reply:
Ok, I see they are using the empty string (""). This relies completely in having the right position, which, given the bioinformatics history is a little bit risky, but anyway.

Reply:
@jrambla Yes; you don't get the "error correction" anchor base. But this is in line w/ VRS; and importantly as we have seen when working on "evidence/annotation" style data => there (e.g. ClinVar) you just get information about e.g. what was deleted, not the surrounding sequence either => would have to be fetched etc. (Pinging @rahelp ...)